### ISSUE TITLE
PMID:29167439 Title	Eroded telomeres are rearranged in quiescent ...

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#3285]
1. 
NTR: increased subtelomeric amplification during Quiescence/G0
A cell phenotype in which a segment of the subtelomeric DNA  sequence is  duplicated more frequently than in wild type cells during Quiescence/G0

child of 

FYPO:0001740
Term name | increased gross chromosomal rearrangement
(seems OK as does not specify vegetative?)

2.

NTR: normal telomere length during  Quiescence/G0
A physical cellular phenotype observed during  Quiescence/G0 in which telomere length is normal (i.e. indistinguishable from wild type).

sibling of FYPO:0002687 | normal telomere length

3. loss of telomeric sequence during quiescence/G0
child of
shortened telomeres
synonym Increased senescence in Quiescence/G0
The authors are using this to describe the point where the  telomeres are  totally lost (this isn't replicative senescence clearly because cells aren't cycling!)

4.  decreased G0 to G1 transition

phew that one was difficult for me !

**Comment by ValWood** [comment]
5. 
I also need a term  for "normal telomere structure during Quiescence/G0"
for when there is no rearrangement. But there is shortening. And apparently shortening is "organization" so I can't say  "normal telomere structure during Quiescence/G0"

so I got stuck what to call it

**Comment by mah11** [comment]
Of course I have some questions ...

> loss of telomeric sequence during quiescence/G0
> synonym Increased senescence in Quiescence/G0
> The authors are using this to describe the point where the telomeres are totally lost

I didn't get that impression myself.

For one thing, I don't think they're restricting "senescence" to the point where there's no telomeric sequence left at all.

I also think they are using "replicative senescence" to mean the telomere loss that occurs _before_ they make cells go into G0, not to any further telomere loss that takes place during G0.

(Long & skippable elaboration: The way this paper uses "replicative senescence" seems a little woolly - the first mention seems to refer to the loss of viability seen in ter1Δ cells, as initially reported in PMID:18157149. But later they switch to using telomere shortening as a proxy for repl. senescence, which makes it look a bit as though they're using the "replicative senescence" phrase to describe the shortened telomeres in the gels. It does seem that they use "replicative senescence" for all of the "population doubling" time points, including those where there is still some telomere sequence present.)

tl;dr: I'll leave out the synonym.


> decreased G0 to G1 transition

What does this mean? It doesn't make much sense to say a single cell has decreased occurrence of the transition. Is it delayed, or slower (decreased rate)?

Or is this a population phenotype, in which fewer cells in the population undergo the transition? Or even just "loss of viability during G0" and/or "during G0 to G1 transition"?

We have "inviable cell upon G0 to G1 transition" FYPO:0000238 and a couple of cell morphology during ... terms. Could add more if need be.

> normal telomere structure during Quiescence/G0

Which aspect(s) of the structure is/are normal? DNA sequence, chromatin or other folded structure, ... ?

Note to self: fix to make length is_a morphology but not structure for telomere terms (will be more consistent with PATO and will avoid the problem here)


p.s. suppl. fig 1 says catalytically dead telomerase is trt1-D743A

**Comment by ValWood** [comment]
I thought they were using "replicative senescence"  for the experimental set up. They did some experiments where they  used different numbers of cell divisions before entry into   senescence. I ignored this and only curated the observations that whatever they did 
resulted in "increased subtelomeric amplification during Quiescence/G0"

Then, later they switch from using "replicative senescence" to just  "senescence" to describe some of the phenotypes they observe during senescence. I took this  (eventually) to be synonymous with loss of "telomeric sequence during quiescence/G0" which is why I included it.
I think they are saying that this is a "kind of model" for normal senescence in non dividing cells in human. I'm happy for you to leave it out. I found it kind of confusing...


> decreased G0 to G1 transition

here I was trying to capture the observation that

STEEx are counter selected when cells exit from quiescence.

I don't know the best way to capture this "The Fig. 8a shows the percentage of wild type and ter1Δ cells that are unable to form a visible colony after 1, 4, and 8 days in quiescence. "

but
These observations indicated that those cells were not dead in quiescence but were not capable to grow. 

any suggestions?

> normal telomere structure during Quiescence/G0

here they don't have the rearrangements, but they shorten.... there is probably an obvious way to do this ?
I can leave it out.....

> p.s. suppl. fig 1 says catalytically dead telomerase is trt1-D743A

oh thank you!

**Comment by ValWood** [comment]
some edits to my comments above...

**Comment by ValWood** [comment]
You are right  "loss of viability in G0" sounds OK.

**Comment by mah11** [comment]
new terms
  increased subtelomeric DNA amplification during G0 FYPO:0006507
  abnormal telomere morphology FYPO:0006508
  abnormal telomere length FYPO:0006509
  abnormal telomere structure FYPO:0006510
  shortened telomeres during G0 FYPO:0006511
  normal telomere morphology FYPO:0006512
  normal telomere morphology during vegetative growth FYPO:0006513
  normal telomere structure FYPO:0006514
  normal telomere length FYPO:0006515
  normal telomere length during G0 FYPO:0006516
  normal telomere structure during G0 FYPO:0006517
  loss of viability in G0 FYPO:0006518

notes:
- turns out the existing telomere morph/length/etc. terms were all defined as vegetative, so this needed some new grouping terms too. also renamed FYPO:0002254, FYPO:0002155, FYPO:0002687 and descendants to make 'vegetative' explicit. phew.

- all terms with 'G0' in names have 'quiescence' in exact synonyms

- I put FYPO:0006 under FYPO:0001178 because prolonged nitrogen starvation puts pombe into G0. Let me know if I should change that parentage.

edit file: 624241bb5ea36d807028f32008966c4537678811
release: 0a472de1ef2061633019526ba2aab2df216964fb
